templates/cicd/pipeline_cfn.yml

```diff
@@ -224,13 +224,16 @@ Resources:
               - Name: SCCheckoutArtifact
             OutputArtifacts:
               - Name: BuildOutput
-        {{$length := len .Stages}}{{if gt $length 0}}
-        {{range $stage := .Stages}}
-        # {{$stage.Name}} is the same as the name of the environment
-        # this pipeline stage deploys to
+{{$length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}
         - Name: DeployTo-{{$.ProjectName}}-{{$stage.Name}}
-          Actions:{{range $app := $stage.LocalApplications}}
-            # TODO: #220 Add manual approval action if this stage is a prod stage
+          Actions:{{if $stage.Prod}}
```

Comment:
I might be wrong but the `pipeline init` command has a `--enable-cd` flag, should we be checking if that's false before creating this action?

Reply:
That flag will set `stage.Prod`, so essentially we are already checking it.